api/specs.proto

```diff
@@ -101,6 +102,7 @@ message TaskSpec {
 	oneof runtime {
 		NetworkAttachmentSpec attachment = 8;
 		ContainerSpec container = 1;
+		CustomSpec custom = 10;
```

Comment:
We thought about that and ruled in favor for a wrapper message for a few reason, but still open for debate.

1) the Url field of the Any type is supposed to represent the encoding, not the "type"

https://github.com/google/protobuf/blob/master/src/google/protobuf/any.proto#L113
```
  // A URL/resource name whose content describes the type of the
  // serialized protocol buffer message.
```

2) The type in the URL won't be "friendly", e.g., it's going to be a long, namespaced versioned type. The type in the spec will be akin to a top-level type (e.g. "plugin", or "unikernel")

3) By wrapping it into a `*Spec`, we leave ourselves open to provide additional supporting fields (version? controller name? whatever)

Thoughts?